docs/ko/how-to_guides/ThanoSQL_connecting/rest_api_thanosql_tables.md

```diff
@@ -254,15 +254,15 @@ In the following example lets pretend we want to alter the table object below:
         }
     }'
     ```
-  If no `schema_name` query parameter is provided, the parameter defaults to the public schema. 
+  `schema` 제공되지 않으면 기본적으로 public 스키마로 설정됩니다.
 
 
 ## **`POST` /table/{table_name}**
 
-Use this method to execute the CREATE TABLE operation. In order to create the table you simply pass in a database object as a body with the `table_name` and `schema_name` as query params. 
+이 메서드를 사용하여 CREATE TABLE 작업을 실행합니다. 테이블을 생성하려면 데이터베이스 객체를 본문으로 전달하고 쿼리 매개변수로 `table_name` 및 `schema`을 전달하기만 하면 됩니다.
```

Comment:
```suggestion
이 메서드를 사용하여 CREATE TABLE 작업을 실행합니다. 테이블을 생성하려면 테이블 객체를 본문으로 전달하고 쿼리 매개변수로 `table_name` 및 `schema`을 전달합니다.
```
Why there are so many 'database object' in Table APIs?

Reply:
didn't realize it.. I will edit the en version as well